### PR TITLE
Feature/var2vcf_vardict

### PR DESCRIPTION
**Pull request by hassanfa**
Added two modules and two tests for two commands from varDict's package. These modules convert variant table output of the main command ```vardict``` to vcf for paired and single mode data: ```var2vcf_paired.pl``` and ```var2vcf_valid.pl``` respectively.

### CODE REVIEW

File: lib/MIP/Program/Variantcalling/Vardict_var2vcf_paired.pm [hunk 1]
```diff
@@ -0,0 +1,121 @@
+package MIP::Program::Variantcalling::Vardict_var2vcf_paired;
```
Comment (henrikstranneheim) [inline lib/MIP/Program/Variantcalling/Vardict_var2vcf_paired.pm]: Add this as another sub in the Vardict.pm

File: lib/MIP/Program/Variantcalling/Vardict_var2vcf_paired.pm [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+            stderrfile_path        => $stderrfile_path,
+            stderrfile_path_append => $stderrfile_path_append,
+            stdoutfile_path        => $stdoutfile_path,
+        }
+      );
+
+    unix_write_to_file(
+        {
+            FILEHANDLE   => $FILEHANDLE,
+            commands_ref => \@commands,
```
Comment (henrikstranneheim) [inline lib/MIP/Program/Variantcalling/Vardict_var2vcf_paired.pm]: Order AB

File: lib/MIP/Program/Variantcalling/Vardict_var2vcf_single.pm [hunk 3]
```diff
@@ -0,0 +1,121 @@
+package MIP::Program::Variantcalling::Vardict_var2vcf_single;
```
Comment (henrikstranneheim) [inline lib/MIP/Program/Variantcalling/Vardict_var2vcf_single.pm]: Same here! Add as sub Vardict_var2vcf_single in the Vardict.pm

File: lib/MIP/Program/Variantcalling/Vardict_var2vcf_single.pm [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+            stderrfile_path        => $stderrfile_path,
+            stderrfile_path_append => $stderrfile_path_append,
+            stdoutfile_path        => $stdoutfile_path,
+        }
+      );
+
+    unix_write_to_file(
+        {
+            FILEHANDLE   => $FILEHANDLE,
+            commands_ref => \@commands,
```
Comment (henrikstranneheim) [inline lib/MIP/Program/Variantcalling/Vardict_var2vcf_single.pm]: Order AB

**Comment by hassanfa** [comment]
@henrikstranneheim requested updates are added.

**Comment by henrikstranneheim** [comment]
👍

**Pull request merged**